src/coord/Coord.cc

```diff
@@ -68,14 +66,20 @@ CoordSystemMap const getCoordSystemMap() {
     return idMap;
 }
 
-/// Calculate angular distance between two points using the Haversine formula
-///
-/// Besides the differences between the two coordinates, we also input the
-/// cosine of the two declinations, so as to be thrifty with the use of trig functions.
-afwGeom::Angle haversine(afwGeom::Angle const dAlpha, /// Difference between RAs (or longitudes), alpha1-alpha2
-                         afwGeom::Angle const dDelta, /// Difference between Decs (or latitudes), delta1-delta2
-                         double const cosDelta1, /// Cosine of the first Dec, cos(delta1)
-                         double const cosDelta2 /// Cosine of the second Dec, cos(delta2)
+/** Calculate angular distance between two points using the Haversine formula
+ *
+ * Besides the differences between the two coordinates, we also input the
+ * cosine of the two declinations, so as to be thrifty with the use of trig functions.
+ *
+ * @param dAlpha Difference between RAs (or longitudes), alpha1-alpha2
+ * @param dDelta Difference between Decs (or latitudes), delta1-delta2
+ * @param cosDelta1 Cosine of the first Dec, cos(delta1)
+ * @param cosDelta2 Cosine of the second Dec, cos(delta2)
+ */
+afwGeom::Angle haversine(afwGeom::Angle const dAlpha,
+                         afwGeom::Angle const dDelta,
+                         double const cosDelta1,
+                         double const cosDelta2
```

Comment:
I suppose it's less about anonymous namespaces and more about whether the documentation should be limited to public functions.  I'd be okay with documenting these with a `@internal` tag or something similar, but I don't like the idea of functions that can't be called outside a source file appearing in the documentation.